Core/include/Acts/Propagator/detail/SteppingHelper.hpp

```diff
@@ -28,61 +26,53 @@ namespace Acts::detail {
 /// returns the status of the intersection to trigger onSurface in case
 /// the surface is reached.
 ///
+/// @tparam stepper_t The type of stepper used for the propagation
+///
+/// @param stepper [in] The stepper in use
 /// @param state [in,out] The stepping state (thread-local cache)
 /// @param surface [in] The surface provided
+/// @param index [in] The surface intersection index
+/// @param direction [in] The propagation direction
 /// @param boundaryTolerance [in] The boundary check for this status update
+/// @param surfaceTolerance [in] Surface tolerance used for intersection
+/// @param stype [in] The step size type to be set
+/// @param logger [in] A @c Logger instance
 template <typename stepper_t>
 Acts::IntersectionStatus updateSingleSurfaceStatus(
     const stepper_t& stepper, typename stepper_t::State& state,
-    const Surface& surface, std::uint8_t index, Direction navDir,
+    const Surface& surface, std::uint8_t index, Direction direction,
     const BoundaryTolerance& boundaryTolerance, double surfaceTolerance,
-    const Logger& logger) {
+    ConstrainedStep::Type stype, const Logger& logger) {
```

Comment:
I suppose there's no point in defaulting this? What values does it take de-factor?